core/primitives-core/src/account.rs

```diff
@@ -199,6 +199,9 @@ pub struct FunctionCallPermission {
     #[serde(with = "option_u128_dec_format")]
     pub allowance: Option<Balance>,
 
+    // this isn't an AccountId because already existing records in testnet genesis have invalid
```

Comment:
```suggestion
    // This isn't an AccountId because already existing records in testnet genesis have invalid
```

I've found that the rule "make each comment a proper sentence/paragrapth, with correct capitalization and punctuation" improved quality of my comments. It's not that capital letters matter per se, its' just that when you set up to write a proper English sentence, and not a "mere comment", you tend to express yourself more clearly. 

Obviously not important :)